README.md

```diff
@@ -3,18 +3,12 @@
 
 Charts for the [Consumer Financial Protection Bureau](https://cfpb.github.io/).
 
-## Dependencies
+## Demo
```

Comment:
Honest question: why did we remove the list of dependencies and instructions for how to install it non-locally?

Reply:
IMHO it doesn't add any value and it clutters the README making it harder to get started with the project. All the dependencies are installed by the setup script and listed in package.json for the curious.